packages/shared/comms/interface/index.ts

```diff
@@ -1,53 +1,50 @@
 import { Position } from './utils'
 import {
   UserInformation,
-  Package,
   ChatMessage,
   ProfileVersion,
   BusMessage,
   VoiceFragment,
   ProfileResponse,
-  ProfileRequest
+  ProfileRequest,
+  Package
 } from './types'
-import { Stats } from '../debug'
-import { Realm } from 'shared/dao/types'
-import { Profile } from 'shared/types'
 import { EncodedFrame } from 'voice-chat-codec/types'
+import { Emitter } from 'mitt'
+import { Avatar } from '@dcl/schemas'
+
+export type CommsEvents = {
+  initialMessage: Package<UserInformation>
+  sceneMessageBus: Package<BusMessage>
+  chatMessage: Package<ChatMessage>
+  profileMessage: Package<ProfileVersion>
+  position: Package<Position>
+  voiceMessage: Package<VoiceFragment>
+
+  // move the following to GlobalMessages+RPC instead of RoomConnection
+  profileResponse: Package<ProfileResponse>
+  profileRequest: Package<ProfileRequest>
+
+  DISCONNECTION: any
+}
 
-export interface WorldInstanceConnection {
-  stats: Stats | null
-
-  // handlers
-  sceneMessageHandler: (alias: string, data: Package<BusMessage>) => void
-  chatHandler: (alias: string, data: Package<ChatMessage>) => void
-  profileHandler: (alias: string, identity: string, data: Package<ProfileVersion>) => void
-  positionHandler: (alias: string, data: Package<Position>) => void
-  voiceHandler: (alias: string, data: Package<VoiceFragment>) => void
-  profileResponseHandler: (alias: string, data: Package<ProfileResponse>) => void
-  profileRequestHandler: (alias: string, data: Package<ProfileRequest>) => void
-
-  readonly isAuthenticated: boolean
-
-  // TODO - review metrics API - moliva - 19/12/2019
-  readonly ping: number
-  printDebugInformation(): void
-  analyticsData(): Record<string, any>
+export interface RoomConnection {
```

Comment:
@menduz do you want to rename this to CommsConnection or something?

Reply:
No, please keep this PR untouched because it is still changing a lot